content/infrastructure/rippled/installation/capacity-planning.md

```diff
@@ -50,23 +50,19 @@ If you set the `[node_size]` parameter to an invalid value, the [server fails to
 
 The `type` field in the `[node_db]` stanza of the `rippled.cfg` file sets the type of key-value store that `rippled` uses to hold the ledger store.
 
-This setting does not directly configure RAM settings, but the choice of key-value store has important implications for RAM usage because of the different ways these technologies cache and index data for fast lookup.
+For almost all purposes, use `NuDB`. A fast SSD is required. [Learn more](#more-about-using-nudb)
 
-- For most cases, use `NuDB` because its performance is constant even with large amounts of data on disk. A fast SSD is required. [Learn more](#more-about-using-nudb)
-
-- If you are using rotational disks (not recommended) or an unusually slow SSD, use `RocksDB`. You should avoid this setting for production servers. [Learn more](#more-about-using-rocksdb)
+The `RocksDB` setting is available for legacy purposes, but is generally not recommended. [Learn more](#more-about-using-rocksdb)
 
 The example `rippled-example.cfg` file has the `type` field in the `[node_db]` stanza set to `NuDB`.
 
 #### More About Using RocksDB
 
-[RocksDB](https://rocksdb.org/docs/getting-started.html) is an persistent key-value store built into `rippled`.
-
-**Caution:** As of late 2021, the total size of the ledger has grown large enough that servers using RocksDB often struggle to maintain sync with the Mainnet. Large amounts of RAM can help, but you should generally use NuDB instead.
+[RocksDB](https://rocksdb.org/docs/getting-started.html) is an persistent key-value store built into `rippled`. **Support for RocksDB is considered legacy.** Servers using RocksDB usually struggle to maintain sync with the Mainnet. Generally, you should use NuDB instead.
```

Comment:
Instead of "usually struggle":
Servers using RocksDB tend struggle to maintain sync with the Mainnet as database size increases. A few hundred GB seem to be the upper limit of stability, but this is dependent on memory size. NuDB, on the other hand, uses a relatively stable amount of system memory regardless of database size.

Reply:
Rephrased the sentence slightly to add the idea that memory requirements for a large database are the reason for RocksDB struggling.